ably/types/options.py

```diff
@@ -12,7 +12,7 @@ class Options(AuthOptions):
     def __init__(self, client_id=None, log_level=0, tls=True, rest_host=None,
                  realtime_host=None, port=0, tls_port=0, use_binary_protocol=True,
                  queue_messages=False, recover=False, environment=None,
-                 http_open_timeout=None, http_request_timeout=None,
+                 http_open_timeout=None, http_request_timeout=None, realtime_request_timeout=None,
```

Comment:
I think we also need to add the kwarg and docstring for this option to the `AblyRealtime` constructor